install.yaml

```
@@ -29,7 +29,7 @@ post_install_actions:
     {{ $project_tld := "ddev.site" }}
     {{ if .DdevGlobalConfig.project_tld }}{{ $project_tld = .DdevGlobalConfig.project_tld }}{{ end }}
     {{ if .DdevProjectConfig.project_tld }}{{ $project_tld = .DdevProjectConfig.project_tld }} {{ end }}
-    {{ $novarnish_hostnames := print "novarnish." .DdevProjectConfig.name "." $project_tld  }}
+    {{ $novarnish_hostnames := print "novarnish.$DDEV_HOSTNAME" }}
```

I found a way to escape `$` with `\\$`:

```suggestion
    {{ $novarnish_hostnames := print "novarnish.\\$DDEV_HOSTNAME" }}
```